nemo_text_processing/text_normalization/hi/clean_eval_data.py

```diff
@@ -0,0 +1,342 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
how does this file fit into the overall grammar?

Reply:
I’m unsure how this file fits into the overall grammar. Should it be retained, or would it be better to remove it?

Reply:
if you're unsure, maybe try removing it and running tests. if everything passes without it, commit and push the deletion

Reply:
Done